### PR TITLE
Feature/improve validate run mode

### PR DESCRIPTION
**Pull request by rickbijkerk**
Change the validation implementation to depend on a locally provided manifest file
simplified code a bit

### CODE REVIEW

File: cmd/validate.go [hunk 1]
```diff
@@ -17,36 +17,36 @@ import (
...
 	}
 
-	remoteLoader, err := persistedoperations.RemoteLoaderFromConfig(cfg.PersistedOperations, log)
-	if err != nil && !errors.Is(err, persistedoperations.ErrNoRemoteLoaderSpecified) {
-		log.Warn("Error initializing remote loader", "err", err)
-	}
-
-	po, err := persistedoperations.NewPersistedOperations(log, cfg.PersistedOperations, persistedoperations.NewLocalDirLoader(cfg.PersistedOperations, log), remoteLoader)
+	// Load the persisted operations from the local dir into memory
+	persistedOperations, err := persistedoperations.NewPersistedOperations(log, cfg.PersistedOperations, persistedoperations.NewLocalDirLoader(cfg.PersistedOperations, log), nil)
```
Comment (ldebruijn) [inline cmd/validate.go:28]: will this not throw an NPE on the `nil` for the remoteLoader?

File: docs/README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Please see each section for in-depth documentation
 Protect supports various running modes for different needs and purposes.
 
 * `serve` runs as an HTTP proxy protection your GraphQL during runtime. Check out the [Deployment Options](#run) section for more configuration options
-* `validate` runs as a CLI tool, validating your Peristed Operations against your schema and configured protections
+* `validate` runs as a CLI tool, validating your Peristed Operations against your schema and configured protections [example run config](./validate-config-example.yml)
```
Comment (ldebruijn) [inline docs/README.md:14]: Maybe make it explicit that its not a separate config for the validate, but that some options do not have any effect during validations?

File: cmd/validate.go (hunk 1 above)
Comment (rickbijkerk) [reply cmd/validate.go:28]: nope theres a nil check built in somewhere deeper in the code

File: docs/README.md (hunk 2 above)
Comment (rickbijkerk) [reply docs/README.md:14]: ah yea good point

**Pull request merged**